File: output/llama3.1-8b/avatar/Python/Java/atcoder_ABC119_B.java
import java.util.Scanner;

public class atcoder_ABC119_B {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int N = Integer.parseInt(scanner.nextLine());
        double[] x = new double[N];
        for (int i = 0; i < N; i++) {
            String input = scanner.nextLine();
            if (input.contains("JPY")) {
                input = input.replace("JPY", "* 1.0");
            } else if (input.contains("BTC")) {
                input = input.replace("BTC", "* 380000.0");
            }
            x[i] = Double.parseDouble(input);
        }
        double sum = 0;
        for (double d : x) {
            sum += d;
        }
        System.out.println(sum);
    }
}

File: output/llama3.1-8b/avatar/Java/Python/atcoder_ABC119_B.py
import sys

def solve(x, u):
    return sum(x[i] * (380000 if u[i] == 'JPY' else 1) for i in range(len(x)))

N = int(input())
x = [float(input()) for _ in range(N)]
u = [input() for _ in range(N)]

print(solve(x, u))